<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization_expert</class>
 <widget class="QMainWindow" name="Authorization_expert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>148</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>432</width>
    <height>148</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>432</width>
    <height>148</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Авторизация эксперта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_name_expert">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите ваше ФИО:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_leader">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>За вами закреплен руководитель:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name_leader">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_expert_category">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваша категория на данный момент:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_expert_category">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Далее</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_select_expert">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
